resources/views/_snippets/styles.blade.php

```diff
@@ -23,8 +23,8 @@
 
 <script>
     document.addEventListener('DOMContentLoaded', (event) => {
-        document.querySelectorAll('pre code.language-bash, pre code.language-php, pre code.language-yaml, pre code.language-diff, pre code.language-json').forEach((block) => {
-            hljs.highlightBlock(block);
+        document.querySelectorAll('pre code.language-bash, pre code.language-php, pre code.language-yaml, pre code.language-diff, pre code.language-json').forEach((element) => {
+            hljs.highlightElement(element);
```

Comment:
Thanks for addressing this :clap: . 
I've seen some error in console log, but didn't understand what to do about it